client/src/js/users/components/__tests__/Create.test.js

```diff
@@ -1,71 +1,182 @@
+jest.mock("connected-react-router");
+
 import * as actions from "../../actions";
-import CreateUserContainer, { CreateUser } from "../Create";
+import CreateUserContainer, { CreateUser, mapStateToProps, mapDispatchToProps } from "../Create";
+import { push } from "connected-react-router";
 
 describe("<CreateUser />", () => {
     let props;
-    let wrapper;
+    let state;
+    let hasError;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['hasError' is defined but never used. (no-unused-vars)](https://app.codacy.com/app/virtool/virtool/pullRequest?prid=4652072)